CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
Thanks, @jcajka. Yes, I confirm that I run `make test-unit` as non-root user and I do not set any environment variables, expecting `Makefile` and `hack/*` and whatnot to take care of that.

If it's easy for you to do the build for Fedora 25, I'll be happy to test that.